src/main/resources/db/changelog/v.0.3/db.changelog-skill.yaml

```
@@ -0,0 +1,64 @@
+databaseChangeLog:
+  - changeSet:
+      id: 12042021-add-type-skills
+      author: maksim_maevsky
+      dbms: postgresql
+      sql: CREATE TYPE SKILL as ENUM ('JAVA', 'JS', 'C_SHARP', 'C_PLUS_PLUS', 'QA', 'DEV_OPS', 'SWIFT', 'GO', 'PHP', 'PYTHON', 'KOTLIN', 'RUST');
+
+  - changeSet:
+      id: 12042021-add-type-internship-skill
+      author: maksim_maevsky
+      changes:
+        - createTable:
+            tableName: internship_skill
+            columns:
+              - column:
+                  name: skill_name
+                  type: SKILL
+                  constraints:
+                    nullable: false
```

```suggestion
                    nullable: true
```

skill_name can't exist without is_inship_id

---

done

---

maybe false is right?

---

Looks like false is correct